### PR TITLE
♻️ Refactor code

### PR DESCRIPTION
**Pull request by J-Bannach**
- Remove unnecessary code
- Change routes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ dist
 dist-ssr
 *.local
 .eslintcache
-.env
\ No newline at end of file
+.env
+.vscode
```
Comment (lmachens) [inline .gitignore:8]: Good job ignoring `.vscode`, but this effect only upcoming changes.
`.vscode` is already in your git index and needs to be removed.
Maybe this helps:
https://stackoverflow.com/questions/2223308/how-to-remove-a-file-from-the-staging-area-index-cache-in-git

I expect to see all `.vscode` files removed in this pull request.

File: src/app/pages/LandingPage/LandingPage.tsx [hunk 2]
```diff
@@ -1,21 +1,26 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import GridImage from '../../components/GridImage/GridImage';
 import styles from './LandingPage.module.css';
 
 function LandingPage(): JSX.Element {
+  const history = useHistory();
+  setTimeout(() => {
+    history.push('/home');
+  }, 4000);
```
Comment (lmachens) [inline src/app/pages/LandingPage/LandingPage.tsx:10]: This works well, but there could be an issue if the component re-renders. You start a new timeout on each render, thus, make sure to only start a single timeout.
This is possible with a `useEffect`:

```suggestion
  useEffect(() => {
    setTimeout(() => {
      history.push('/home');
    }, 4000);
  }, []);
```

**Review by lmachens — COMMENTED** [review]
Good job

**Review by SGierszewski — COMMENTED** [review]
Looks good :)

File: src/app/pages/LandingPage/LandingPage.tsx (hunk 2 above)
Comment (J-Bannach) [reply src/app/pages/LandingPage/LandingPage.tsx]: Thanks, I didn't think about the re-rendering.  Changed the code now.

File: .gitignore (hunk 1 above)
Comment (J-Bannach) [reply .gitignore:8]: Thanks for the heads-up. Had to use `git rm -r --cached <name>` though.

**Pull request merged**